### PR TITLE
Fix parsing of colors with decimal alpha

### PR DESCRIPTION
**Pull request by kraynel**
Fix issue https://github.com/jsdom/jsdom/issues/2639. 
Color with decimal alpha values were not recognized.

### CODE REVIEW

**Comment by jsakas** [comment]
@kraynel thanks for the PR. Reviewed and it looks solid, thanks for including the tests!

File: lib/utils/colorSpace.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  if (hue < 0) hue += 6;
+  if (hue >= 6) hue -= 6;
+
+  if (hue < 1) return (t2 - t1) * hue + t1;
+  else if (hue < 3) return t2;
+  else if (hue < 4) return (t2 - t1) * (4 - hue) + t1;
+  else return t1;
+};
+
+// https://www.w3.org/TR/css-color-4/#hsl-to-rgb
```
Comment (TimothyGu) [inline lib/utils/colorSpace.js]: For future PRs, please make sure to use the latest editor's draft, as found on https://drafts.csswg.org/. For this case, it'll be https://drafts.csswg.org/css-color-4/#hsl-to-rgb.

**Comment by domenic** [comment]
@jsakas would you mind doing a release that includes this?

**Comment by jsakas** [comment]
@domenic will be publishing v2.1.0 this afternoon.

**Pull request merged**